modules/openy_features/openy_node/modules/openy_node_activity/config/install/core.entity_form_display.node.activity.default.yml

```diff
@@ -4,9 +4,11 @@ dependencies:
   config:
     - field.field.node.activity.field_activity_category
     - field.field.node.activity.field_activity_description
+    - field.field.node.activity.field_meta_tags
```

Comment:
I'm not sure that this CT needs meta tags. There is no individual page for it. It's available only in listing as filter.